### PR TITLE
Car/garden singleton

### PR DESCRIPTION
**Pull request by TheRedPanda17**

### CODE REVIEW

File: myning/chapters/enter_garden.py [hunk 1]
```diff
@@ -40,7 +39,7 @@ def get_seeds():
                 seeds.remove(seed)
                 player.inventory.add_item(seed)
                 FileManager.save(seed)
-            FileManager.save(player)
+            FileManager.multi_save(player, garden)
```
Comment (TheRedPanda17) [inline myning/chapters/enter_garden.py:42]: This will look less annoying when we pull inventory out. It should be the next itme.

File: Makefile [hunk 2]
```diff
@@ -27,5 +27,11 @@ restart:
 lint: isort black
 	@true
 
+isort:
+	isort . --check --diff;
```
Comment (rj-lee) [inline Makefile:31]:
```suggestion
	isort . --check --diff;
```

File: Makefile [hunk 3]
```diff
@@ -27,5 +27,11 @@ restart:
 lint: isort black
 	@true
 
+isort:
+	isort . --check --diff; 
+
+black:
+	black . --check;
```
Comment (rj-lee) [inline Makefile:34]:
```suggestion
	black . --check
```

**Review by rj-lee — DISMISSED** [review]

File: Makefile (hunk 3 above)
Comment (TheRedPanda17) [reply Makefile:34]: Why no semicolon on this one?

Comment (rj-lee) [reply Makefile:34]: i meant to remove the semicolon on the other one as well, i don't think it's necessary

Comment (TheRedPanda17) [reply Makefile:34]: I'll check to make sure when I'm back

**Review by rj-lee — APPROVED** [review]

**Pull request merged**